notebook/scale-test.py

```diff
@@ -130,7 +130,7 @@ def get_cookie(session, name):
 
     print(f'successes: {len(times)} / {n} = {len(times) / n}')
     print(f'mean time: {sum(times) / n}')
-    print(f'max time: {max(times)}')
+    print(f'quantiles min/50%/95%/99%/max:: {np.quantile(times, [0.0, .5, .95, .99, 1.0])}')
```

Comment:
Did you mean to have the double colon?

Reply:
no, my keyboard inserts between 0 and 3 characters on each key press.